indico/modules/events/papers/templates/management/cfp_actions/ended.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'events/papers/management/cfp_actions/_base.html' %}
+
+{% block label %}
+    {% trans -%}
+        The call for papers is closed
+    {%- endtrans %}
+{% endblock %}
+
+{% block text %}
+    {% set tz = event.timezone %}
+    {% trans date=cfp.end_dt|format_date(timezone=tz), time=cfp.end_dt|format_time(timezone=tz) -%}
+        Closed on <strong>{{ date }}</strong> at <strong>{{ time }}</strong> ({{ tz }}).
+    {%- endtrans %}
+{% endblock %}
+
+{% block buttons %}
+    {{ super() }}
+    <div class="group">
+        <a href="#" class="i-button"
+           data-href="{{ url_for('.open_papers_call', event) }}"
+           data-method="POST"
+           data-update="#paper-page"
+           data-replace-update
```

Comment:
Not a strong opinion but I personally put the attributes with no value at the bottom of the list (same for [this](https://github.com/indico/indico/pull/2751/files#diff-9d5d15c6e084cdd07fc7a21641b78312R19).